styles.css

```diff
@@ -0,0 +1,108 @@
+body {
+	align-items: center;
+	color: white;
+	background-color: rgba(58, 2, 82, 0.447);
+	display: flex;
+	flex-direction: column;
+	font-family: "Notable", sans-serif;
+}
+
+#page-break {
+	border: 100px solid rgb(63, 61, 61);
+	display: inline-flex;
+	font-size: 40px;
+	text-align: center;
+}
+
+#page-break-1 {
+	border: 5px solid rgb(63, 61, 61);
+}
+
+#header {
+	background-color: darkred;
+}
+
+#features {
+	background-color: greenyellow;
+}
+
+.footer-form-container {
+	align-items: center;
+	display: inline-flex;
+	flex-direction: row;
+}
+
+#home-img-container {
+	display: flex;
+	justify-content: space-around;
+}
+
+#main {
+	display: flex;
+	justify-content: space-around;
+}
+
+.purple-btn {
+	color: white;
+	width: 100px;
+	background-color: purple;
+	border-color: black;
+	cursor: pointer;
+}
+
+.purple-btn:hover {
+	background-color: rgb(63, 61, 61)
+}
+
+.content-width {
+	min-width: 700px;
+	width: 70%;
+}
+
+.home-img-wrapper {
+	width: 30%;
+	height: auto;
+}
+
+.image {
+	width: 100%;
+	height: auto;
+}
+
+.dark-gray-btn {
+	color: white;
+	background-color: rgb(63, 61, 61);
+	cursor: pointer;
+}
+
+.dark-gray-btn:active {
+	background-color: purple;
+}
+
+.dark-gray-btn:focus {
+	color: red;
+}
+
+.column {
+	display: flex;
+	flex-direction: column;
+	width: 30%;
+}
+
+.row {
+	display: flex;
+	flex-direction: row;
+}
+
+h1 {
+	font-size: 60px;
+	font-family: "Notable", sans-serif;
+	text-align: center;
+}
+
+h3 {
+	text-align: center;
+	font-size: 30px;
+	font-family: "Notable", sans-serif;
+	page-break-after: auto|always|avoid|left|right|initial|inherit;
+}
```

Comment:
These styles are being over written by the CSS written within your HTML pages. One way to avoid that is to be more specific with your labeling, Give you tags specific ids or classes and reference those for the CSS rather then the `<h1>`